app/assets/javascripts/TT_init.js

```diff
@@ -77,6 +83,8 @@ $(document).ready(function() {
     event.which == 32 && startCallback(event);
     // ESC key
     event.which == 27 && resetCallback(event);
+    // PAUSE key
+    event.which == 19 && pauseCallback(event);
```

Comment:
I think this makes a lot of sense, but for mac users it would be a little bit unaccessible, see https://en.wikipedia.org/wiki/Break_key.